packages/mgt-components/src/components/mgt-people-picker/mgt-people-picker.ts

```diff
@@ -502,6 +502,11 @@ export class MgtPeoplePicker extends MgtTemplatedTaskComponent {
    */
   @state() private _foundPeople: IDynamicPerson[];
 
+  /**
+   * Checks if the input has maximum selected people in single select mode.
+   */
+  @state() private _hasMaxSelections = false;
```

Comment:
I think that the original implementation of get hasMaxSelections was fine, I'm not sure what's gained here by making this a state decorated reactive property.

Reply:
It was in my implementation idea of setting hasMaxSelections based on selectedPeople, default selectedUsers and groups and I didn't want to have a getter and setter. However, as they all end up as selectedPeople, I have reverted this.